src/common/acl_util.c

```diff
@@ -423,6 +451,20 @@ daos_ace_to_str(struct daos_ace *ace, char *buf, size_t buf_len)
 		rc = write_char(&pen, PERM_READ_CH, &remaining_len);
 	if (perms & DAOS_ACL_PERM_WRITE)
 		rc = write_char(&pen, PERM_WRITE_CH, &remaining_len);
+	if (perms & DAOS_ACL_PERM_CREATE_CONT)
+		rc = write_char(&pen, PERM_CREATE_CONT_CH, &remaining_len);
+	if (perms & DAOS_ACL_PERM_DEL_CONT)
+		rc = write_char(&pen, PERM_DEL_CONT_CH, &remaining_len);
+	if (perms & DAOS_ACL_PERM_GET_PROP)
+		rc = write_char(&pen, PERM_GET_PROP_CH, &remaining_len);
+	if (perms & DAOS_ACL_PERM_SET_PROP)
+		rc = write_char(&pen, PERM_SET_PROP_CH, &remaining_len);
+	if (perms & DAOS_ACL_PERM_GET_ACL)
+		rc = write_char(&pen, PERM_GET_ACL_CH, &remaining_len);
+	if (perms & DAOS_ACL_PERM_SET_ACL)
+		rc = write_char(&pen, PERM_SET_ACL_CH, &remaining_len);
+	if (perms & DAOS_ACL_PERM_SET_OWNER)
+		rc = write_char(&pen, PERM_SET_OWNER_CH, &remaining_len);
```

Comment:
these are the types of things I like to use FOREACH macros for.

Reply:
I added a macro to simplify this in my next PR. FOREACH seemed like overkill in this case, but I did add a macro to limit the repetition.